src/plugins/hetero/src/compiled_model.hpp

```diff
@@ -21,7 +21,10 @@ class CompiledModel : public ov::ICompiledModel {
                   const std::shared_ptr<const ov::IPlugin>& plugin,
                   const Configuration& cfg);
 
-    CompiledModel(std::istream& model, const std::shared_ptr<const ov::IPlugin>& plugin, const Configuration& cfg);
+    CompiledModel(std::istream& model,
+                  const std::shared_ptr<const ov::IPlugin>& plugin,
+                  const Configuration& cfg,
+                  const bool loaded_from_cache = false);
```

Comment:
do we need a default argument? I suppose users need explicitly specify whether it's from model cache or not

Reply:
Agree!